src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -417,6 +417,29 @@ public int MicrobeHexSize
         (IsMulticellularEditor ? Constants.MULTICELLULAR_EDITOR_COST_FACTOR : 1.0f) *
         Editor.CurrentGame.GameWorld.WorldSettings.MPMultiplier;
 
+    public static void InitOrganelleModel(SceneDisplayer organelleModel, OrganelleTemplate organelle)
```

Comment:
This doesn't literally init the model builder, this just sets the position and scale for it.